<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>108</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Seach &amp; replace</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QPushButton" name="searchDialog_replaceAllButton">
     <property name="text">
      <string>Replace all</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="ocurrencesCounterLabel">
     <property name="text">
      <string>0/0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QPushButton" name="searchDialog_replaceButton">
     <property name="text">
      <string>Replace</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QLineEdit" name="searchDialog_replaceLineEdit">
     <property name="placeholderText">
      <string>Replace with...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QPushButton" name="gobSwapTextButton">
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Swap</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QLineEdit" name="seachDialog_searchLineEdit">
     <property name="placeholderText">
      <string>Find...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QPushButton" name="searchDialog_searchButton">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>seachDialog_searchLineEdit</tabstop>
  <tabstop>searchDialog_replaceLineEdit</tabstop>
  <tabstop>searchDialog_replaceButton</tabstop>
  <tabstop>searchDialog_replaceAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
